src/mistral_common/protocol/instruct/normalize.py

```diff
@@ -36,6 +36,7 @@ class InstructRequestNormalizer(
     """
 
     system_prompt_in_begin: bool = False
+    allow_tool_call_and_content: bool = False
```

Comment:
I like that below you rename `allow_tool_call_and_content` with a leading `_` to mark it private. Can we also do that for these args ? The user is not supposed to handle this right ?

Reply:
Yeah fair